apps/index.html

```diff
@@ -305,6 +305,22 @@
       </div>
 
       <div class="app">
+        <button onclick="install('118d77fa72eba6a685cb493bffb18fae?url=https://github.com/libreboard/libreboard/releases/download/v0.7.1/libreboard.spk')">Install »</button>
+        <h3>LibreBoard</h3>
+        <div class="app-details">
+          Originally by: <a href="https://github.com/yasaricli">Yaşar İçli</a> &amp; <a href="https://github.com/mquandalle">Maxime Quandalle</a><br>
+          License: MIT<br>
+          Code: <a href="https://github.com/libreboard/libreboard">On GitHub</a><br>
+          Updated: Jan 11 2015
+        </div>
+        <p>A “kanban” board similar to Trello that let you organize things in cards, and cards in
+        lists. It supports most features you can expect (labels, comments, markdown formatting...)
+        and is built on <a href="https://meteor.com">Meteor</a>. Sandstorm is officialy supported
+        by the development team so no port was needed.
+        <p><img src="libreboard.png">
```

Comment:
Well, none of the other images have width/height, so we might as well just plan to fix this whole page up later. :)